app/src/main/java/com/rafraph/pnineyHalachaHashalem/myAudio.java

```diff
@@ -65,19 +89,24 @@
     ArrayList<String> sections;
     private String url;
     View view;
-
+    Elements headers;
+    Document doc = null;
+    private Spinner spinner;
     private final Handler handler = new Handler();
-    private TextView header_text;
+
     private String header;
     public ListView listview;
     public String book_name;
+    public TextView txtI;

```

Comment:
Pleave rename **txtI** to some other name that will be easy to understand the role of this variable

Reply:
שיניתי